libs/ui/src/lib/header/index.tsx

```diff
@@ -17,33 +19,29 @@ export interface Props {
    * @default {false}
    */
   useDemoLogo?: boolean;
-  /**
-   * Username to display in header
-   * @type {string}
-   */
-  username?: string;
-  /**
-   * Logout function
-   * @type {() => void;}
-   */
-  onLogout?: () => void;
 }
 
-export const Header: FC<Props> = ({
-  homeUrl,
-  useDemoLogo,
-  username,
-  onLogout,
-}) => {
+export const Header: FC<Props> = ({homeUrl, useDemoLogo}) => {
   const [isDropdownMenuOpen, setIsDropdownMenuOpen] = useState(false);
 
   const openDropdownMenu = () => setIsDropdownMenuOpen(true);
   const closeDropdownMenu = () => setIsDropdownMenuOpen(false);
 
+  const router = useRouter();
```

Comment:
Jeg tenkte å ha dette i header komponent i appen siden den refererer til api/auth/logout som kan være forskjellige fra eventuelt en annen app. Men i etterkant synes jeg det er bedre å ha det slik du har gjort nå og vi kan endre det når vi ser behov for det.

Reply:
Ja, den måten tvinger oss litt på å ha så mye likt som mulig.